packages/client/tests/functional/tracing/tests.ts

```diff
@@ -768,61 +768,64 @@ testMatrix.setupTestSuite(({ provider }, suiteMeta, clientMeta) => {
     })
 
     // Different order of traces between binary and library
-    testIf(getClientEngineType() === ClientEngineType.Library)('should trace the implicit $connect call', async () => {
-      const email = faker.internet.email()
+    testIf(getClientEngineType() === ClientEngineType.Library && getQueryEngineProtocol() !== 'json')(
```

Comment:
How can we unskip this test in the future?

Reply:
I think in this case we can just remove the snapshot - test asserts on the order of specific spans anyway.

Reply:
Did just that, now test is unskipped (for binary engine as well)